packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -470,10 +478,18 @@ export class RunningSummarizer implements IDisposable {
 			let overrideDelaySeconds: number | undefined;
 			let summaryAttempts = 0;
 			let summaryAttemptsPerPhase = 0;
+			// Reducing the default number of attempts to defaultNumberofSummarizationAttempts.
+			const totalAttempts =
+				this.mc.config.getNumber("Fluid.Summarizer.Attempts") ??
+				defaultNumberSummarizationAttempts;
+
+			if (totalAttempts < 1 || totalAttempts > attempts.length) {
```

Comment:
It would be good to default to some reasonable value (maybe defaultNumberSummarizationAttempts) if feature gate gives non-number of negative.